src/pages/ThorChainLP/components/ReusableLpConfirm.tsx

```diff
@@ -0,0 +1,286 @@
+import { ArrowBackIcon } from '@chakra-ui/icons'
+import {
+  Button,
+  Card,
+  CardBody,
+  CardFooter,
+  CardHeader,
+  Center,
+  Flex,
+  IconButton,
+  Skeleton,
+  Stack,
+} from '@chakra-ui/react'
+import type { AssetId } from '@shapeshiftoss/caip'
+import { thorchainAssetId } from '@shapeshiftoss/caip'
+import type { Asset } from '@shapeshiftoss/types'
+import prettyMilliseconds from 'pretty-ms'
+import React, { useMemo } from 'react'
+import { FaPlus } from 'react-icons/fa'
+import { useTranslate } from 'react-polyglot'
+import { Amount } from 'components/Amount/Amount'
+import { AssetIcon } from 'components/AssetIcon'
+import { Row } from 'components/Row/Row'
+import { SlideTransition } from 'components/SlideTransition'
+import { RawText } from 'components/Text'
+import { Timeline, TimelineItem } from 'components/Timeline/Timeline'
+import { getChainAdapterManager } from 'context/PluginProvider/chainAdapterSingleton'
+import { assertUnreachable } from 'lib/utils'
+import { AsymSide, type ConfirmedQuote } from 'lib/utils/thorchain/lp/types'
+import { selectAssetById } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import type { ParsedPool } from '../queries/hooks/usePools'
+import { PoolIcon } from './PoolIcon'
+
+type ReusableLpConfirmProps = {
+  handleBack: () => void
+  handleConfirm: () => void
+  pool?: ParsedPool
```

Comment:
Non-blocking: could use `opportunityId` instead of passing the whole pool - this will be a bit boilerplaty but effectively cleaner - we can "fetch" all pools here (they won't be refetched because something something caching) using the query hook and then select the one pool